src/main/java/org/opentripplanner/routing/core/FareComponent.java

```diff
@@ -15,13 +15,20 @@ public class FareComponent {
     public FeedScopedId fareId;
     public Money price;
     public List<FeedScopedId> routes;
+    public int legIndex;
+    public boolean isTransfer;
```

Comment:
Add short JavaDoc to explain the meaning of `isTransfer`.

Reply:
Did I do that right?

Reply:
Looks good, thanks!